xrayutilities/xrayutilities_default.conf

```diff
@@ -43,7 +43,7 @@ dynhigh = 0
 # of to vectors, which are supposed to be orthogonal)
 epsilon = 1e-8
 
-dbname = elements.db
+dbname = /Users/nl7/work/Materials_Informatics/JILA_experiments/BiFeO3/elements.db
 
```

Comment:
this should be always done in a user specific config file and not in the main one...

Reply:
Indeed.